src/Middleware/Rewrite/src/ApacheModRewrite/RuleBuilder.cs

```diff
@@ -156,7 +157,7 @@ public void AddRule(string rule)
     }
 
     public void AddMatch(
-        ParsedModRewriteInput input,
+        [StringSyntax(StringSyntaxAttribute.Regex)] ParsedModRewriteInput input,
```

Comment:
Revert this.  The attribute is only relevant on strings.

Reply:
yup reverted! :)